internal/pkg/otel/samples/darwin/managed_otlp/platformlogs.yml

```diff
@@ -0,0 +1,67 @@
+receivers:
+  # Receiver for platform specific log files
+  filelog/platformlogs:
+    include: [ /var/log/*.log ]
+    retry_on_failure:
+      enabled: true
+    start_at: end
+    storage: file_storage
+#   start_at: beginning
+
+extensions:
+  file_storage:
+    directory: ${env:STORAGE_DIR}
+
+processors:
+  resourcedetection:
+    detectors: ["system"]
+    system:
+      hostname_sources: ["os"]
+      resource_attributes:
+        host.name:
+          enabled: true
+        host.id:
+          enabled: false
+        host.arch:
+          enabled: true
+        host.ip:
+          enabled: true
+        host.mac:
+          enabled: true
+        host.cpu.vendor.id:
+          enabled: true
+        host.cpu.family:
+          enabled: true
+        host.cpu.model.id:
+          enabled: true
+        host.cpu.model.name:
+          enabled: true
+        host.cpu.stepping:
+          enabled: true
+        host.cpu.cache.l2.size:
+          enabled: true
+        os.description:
+          enabled: true
+        os.type:
+          enabled: true
+
+exporters:
+  # Exporter to print the first 5 logs/metrics and then every 1000th
+  debug:
+    verbosity: detailed
+    sampling_initial: 5
+    sampling_thereafter: 1000
+
+# Exporter to send logs and metrics to Elasticsearch Motel
+  otlp/ingest:
+    endpoint: ${env:ELASTIC_ENDPOINT}
```

Comment:
Regarding the env variable naming, I dont know if we started to differentiate it for the Managed Ingest endpoint yet. @AlexanderWert @mlunadia Any preference/thoughts on this? Should we take this opportunity to decide ?

Reply:
Good point, discussed async with @AlexanderWert and @mlunadia and `ELASTIC_OTLP_ENDPOINT` seems a good solution to differentiate the current Elasticsearch endpoint. changed https://github.com/elastic/elastic-agent/pull/6630/commits/0650f25f99ea887e3797bf01353719d8fd5f9e41